_config.yml

```diff
@@ -0,0 +1,12 @@
+exclude: [
+  'tests',
+  'scripts',
+  'composer.json',
+  'CONTRIBUTING.md',
+  'Dockerfile',
+  'index.js',
+  'LICENSE.md',
```

Comment:
I'd say the LICENSE should be included in the website

Reply:
Yes, I was doubting about it. I will remove it from the exclusions. :+1: